compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/MetalRedrawer.kt

```diff
@@ -345,6 +356,23 @@ internal class MetalRedrawer(
             }
         }
     }
+
+    private fun configureMetalHUDIfNeeded() {
+        if (available(OS.Ios to OSVersion(16))) {
+            val entry = NSBundle
+                .mainBundle
+                .objectForInfoDictionaryKey("MetalHudEnabled") as? NSNumber
```

Comment:
Is it a custom property? Will the name "Compose.MetalHudEnabled" be by the naming style?

Reply:
No, it's the actual plist value required by Apple. User can add this to Plist, no API change in ComposeConfiguration is needed.